pkg/riff/commands/function_create.go

```diff
@@ -144,9 +146,12 @@ func NewFunctionCreateCommand(c *cli.Config) *cobra.Command {
 	opts := &FunctionCreateOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "create",
-		Short:   "<todo>",
-		Example: "<todo>",
+		Use:   "create",
+		Short: "build a function from source",
```

Comment:
Probably deserves a long description. I'd particularly like to see mention of being able to omit the image name in some circumstances.